src/_codux/boards/sidebar/sidebar.board.tsx

```diff
@@ -0,0 +1,13 @@
+import { BrowserRouter } from 'react-router-dom';
+import { createBoard } from '@wixc3/react-board';
+import Sidebar from '../../../components/sidebar';
+
+export default createBoard({
+   name: 'Sidebar',
+   Board: () => (
+      <BrowserRouter>
+         <Sidebar />
+      </BrowserRouter>
+   ),
```

Comment:
`BrowserRouter` is not working in Codux right now, for now boards can handle `MemoryRouter`